src/components/StyleGuide/Colors.stories.mdx

```diff
@@ -91,33 +394,59 @@ export const getColorCards = (cat) => {
 
 # NYPL Colors
 
-| Table of Contents                        |
-| ---------------------------------------- |
-| 1. [Brand Colors](#brand-colors)         |
-| 2. [Section Colors](#section-colors)     |
-| 3. [Utility Colors](#utility-colors)     |
-| 3. [Grayscale Colors](#grayscale-colors) |
-| 4. [Figma Reference](#figma-reference)   |
+| Table of Contents                              |
+| ---------------------------------------------- |
+| 1. [Overview](#overview)                       |
+| 2. [Brand Fills](brand-fills)                  |
+| 3. [NYPL Brand Colors](#nypl-brand-colors)     |
+| 4. [NYPL Section Colors](#nypl-section-colors) |
+| 5. [UI Fills](ui-fills)                        |
+| 6. [Utility Colors](#utility-colors)           |
+| 7. [Grayscale Colors](#grayscale-colors)       |
+| 8. [Figma Reference](#figma-reference)         |
+
+## Overview
+
+The Reservoir Design System (DS) the provides the NYPL branded and approved
+color palette. The NYPL color palette is used to style the DS components and
+is also available for consuming applications in the form of javascript and
+CSS design tokens. Please refer to the [Design Tokens](http://localhost:6006/?path=/story/style-guide-design-tokens--page)
+page for information about consuming and using all style tokens.
+
+Whenever using these colors, designer must ensure that the foreground and
+background colors pass, at a minimum, the `WCAG AA` accessibility guidelines for
+color contrast. For reference, to meet the `WCAG AA` standard, the contrast ratio
+must be at least 4.5:1. To meet the `WCAG AAA` standard, the contrast ratio must
+be at least 7:1.
 
-## Brand Colors
+## Brand Fills
 
-Brand fills are the NYPL primary and secondary colors.
+These colors are connected to the NYPL brand and are not intended to be generic.
+
+### NYPL Brand Colors
```

Comment:
I find it a bit odd that the TOC includes h2s and h3s while it appears as if they are all at the same level. Maybe the TOC for these h3s should skip the number prefix?

Reply:
I was also a little worried about that.  I converted the TOC format to now match the format on the story pages, which allows for better nesting and visual clarity.  I will add new ticket to do the same for the other `Style Guide` pages.